kv/store.go

```diff
@@ -16,6 +16,7 @@ var stores = map[string]func(string) (Store, error){
 	"goleveldb": newGoleveldb,
 	"bbolt":     newBbolt,
 	"badger":    newBadger,
+	"bitcask":   newBitcask,
```

Comment:
Thanks for the contribution! It seems the actual implementation is missing for the new store.

Reply:
sorry 😅
computers are hard

fixed

Reply:
Thanks for fixing it quickly! I'm going to add some basic CI checks to make this kinds of problems easier to spot in the future.